test/e2e/code-examples.spec.js

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from '@playwright/test';
+import { existsSync, statSync } from 'fs';
+
+const groups = ['example', 'example-ssr'];
+
+groups.forEach(path => {
+	test(`Navigate between code samples for "${path}/Bar"`, async ({ page }) => {
+		await page.goto(`${path}/Bar`);
+		// Using has-text here to avoid the ul intercepting the click
+		// https://github.com/microsoft/playwright/issues/12821#issuecomment-1069630780
+		await page.click(`li:has-text('./_components/Bar.svelte')`);
```

Comment:
FYI I had to change the way the click was being created here because the test was failing the `ul` element was intercepting the click for some reason. This solution seemed maybe better than doing `{force: true}`.

Reply:
Good to know, thanks!